weareinreach/inreach-catalog
# Feature/1201524083399270/change name setting

## Description

For "Account Settings Personal" add a drop down to Change the account name

## Asana ticket:
https://app.asana.com/0/1132189118126148/1201524083399270

## Figma design
https://www.figma.com/file/rqCe5GNO7Dibfy5X3hjaRO/%5BDONE---Mar-'20%5D-Account-Settings-Redesign?node-id=161%3A2503

These changes are not reflected in the Figma Designs (they were compiled via slack conversations and referencing localization changes for the General settings sections, see this link for details, https://github.com/asylum-connect/asylumconnect-catalog/issues/117):

## PR Checklist



- [ ] Assign TBD **and** @Alfredo-Moreira as reviewers.
- [ ] localization files updated

## How to Test

1. Create a new account or log in with an existing account
2. go to Account Settings
3. Click on Change Name (to open a drop down form)
4. There are three input fields:
- old name - should auto-populate with the value in the users DB for 'name'. if the DB name field is empty, all whitespace, or contains the generic 'user name' value, the placeholder 'John Smith' will appear
- new name - should show a placeholder 'New name'
- confirm name - should show a placeholder 'New name'
5. complete the input fields
6. click the 'Change Name' button

Provided there are no errors
- the data should be saved to the DB
- the UI will be updated to reflect the new name in the old name field
- both the new name and confirm name fields will be cleared

all fields are required but no checks have been put in place to test for input value length or character type (none were requested)
